application/controllers/ConfigurationController.php

```diff
@@ -139,4 +156,14 @@ protected function activateTab($name = null)
         
         return $this;
     }
+
+    protected function mergeTabs(Tabs $tabs): self
```

Comment:
Please add phpDoc.
Why are you returning $this? You can avoid it or return tabs instead.